pom.xml

```diff
@@ -393,6 +393,7 @@
         <!-- Override source and target from the ASF parent -->
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
```

Comment:
We could also bump the parent POM to 22: https://github.com/apache/maven-apache-parent/blob/apache-22/pom.xml

Reply:
I would not update it for now (just before the release). I think, it is better to bump such versions for the next release. The bump of the `maven-compiler-plugin` was required to support the `release` config.